src/lib/index.ts

```diff
@@ -1,25 +1,16 @@
 // Components
-export { default as Button } from "./Button/Button.svelte";
-export { default as TextBox } from "./TextBox/TextBox.svelte";
-export { default as Navbar } from "../layout/Navbar/Navbar.svelte";
-export { default as Footer } from "../layout/Footer/Footer.svelte";
 export { default as PageSection } from "./PageSection/PageSection.svelte";
-export { default as HyperlinkButton } from "./HyperlinkButton/HyperlinkButton.svelte";
-export { default as IconButton } from "./IconButton/IconButton.svelte";
-export { default as MenuFlyout } from "./MenuFlyout/MenuFlyoutWrapper.svelte";
-export { default as ListViewItem } from "./ListViewItem/ListViewItem.svelte";
 export { default as TreeView } from "./TreeView/TreeView.svelte";
 export { default as HeaderChip } from "./HeaderChip/HeaderChip.svelte";
 export { default as Contributor } from "./Contributor/Contributor.svelte";
 export { default as FeatureCard } from "./FeatureCard/FeatureCard.svelte";
 export { default as ColorSwatch } from "./ColorSwatch/ColorSwatch.svelte";
-export { default as ContentDialog } from "./ContentDialog/ContentDialog.svelte";
 export { default as TerminalCommand } from "./TerminalCommand/TerminalCommand.svelte";
 export { default as Titlebar } from "./Titlebar/Titlebar.svelte";
 export { default as BlogCard } from "./BlogCard/BlogCard.svelte";
-export { default as ProgressRing } from "./ProgressRing/ProgressRing.svelte";
 export { default as Metadata } from "./Metadata.svelte";
 
 // Actions
+export * from "./utils";
 export { default as tilt } from "./tilt";
-export { default as rainbowCanvas } from "./rainbowCanvas";
+export * from "fluent-svelte"; // a very temporary solution
```

Comment:
What does "very temporary" mean here?

Reply:
This is a hack, because Fluent Svelte uses its own components internally, it imports them from `$lib`.
When we use those components, `$lib` is accidentally parsed as our site's `$lib`, not Fluent Svelte's one, and it crashes because it can't find a component.
This re-exports Fluent Svelte's contents from our `$lib`, which solves this issue temporarily until it's fixed within the library.